.github/workflows/release-branch.yml

```diff
@@ -22,20 +22,57 @@ env:
   GH_RUN_NUMBER: ${{ github.run_attempt }}
   GH_BUILD_NUMBER: ${{ github.run_id }}
   JOB_ID: ${{ github.run_id }}
+  PROXY_HOST_BPM: ${{ secrets.E2E_HOST }}
```

Comment:
I think it's incorrect rebase, those fields were cleaned up from the ADF recently

Reply:
This was cleaned recently and in the conflicts between the two as i've fixed the command to match nx 16 they sneaked in, i'll remove them